Project.xml

```diff
@@ -128,7 +132,10 @@
 	<haxedef name="HXCPP_STACK_LINE" if="release" />
 	<haxedef name="HXCPP_STACK_TRACE" if="release" />
 
-	<haxedef name="HXCPP_GC_BIG_BLOCKS" />
+	<!-- Do whatever you want I'm tired uninstall and install everytime -->
+	<certificate path="key.keystore" password="leatherengine" alias="leatherport" alias-password="leatherengine" if="android" unless="debug" />
+
+	<!--<haxedef name="HXCPP_GC_BIG_BLOCKS" />-->

```

Comment:
Why are you commenting out this define

Reply:
it causes memory leaks due big block creation

Reply:
K